stdpopsim/ext/selection.py

```diff
@@ -81,6 +90,7 @@ def __attrs_post_init__(self):
             "g": 0,  # mean
             "n": 1,  # standard deviation
             "w": 0,  # scale
+            "s": -1,  # script types should not be multiplied (lognormal)
```

Comment:
uh-oh: since `-1` means "the last intex" in python, maybe we should instead use `None` to avoid errors?